src/rustdoc-json-types/lib.rs

```diff
@@ -416,6 +418,7 @@ pub enum Type {
     /// `<Type as Trait>::Name` or associated types like `T::Item` where `T: Iterator`
     QualifiedPath {
         name: String,
+        args: Box<GenericArgs>,
```

Comment:
I don't think this should be boxed in the JSON types.

Reply:
So this is needed because I get a recursive type error without the boxing.

Reply:
Ah, ok then.